articles/connections/references/apple-native/_includes/_siwa-prerequisites.md

```diff
@@ -0,0 +1,9 @@
+Before you add support for native SIWA, you'll need:
+
+* An [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/)).
+* A domain (such as `<YOUR CUSTOM DOMAIN>.com`) that you can use and point to (for web apps) and an internet-accessible server where you will run the app, and that responds on behalf of this domain. You will also need to configure this server with a TLS certificate (Apple won't accept unsecured HTTP connections). Lastly, to use the Email Relay Service (for web apps only), you will need to configure your domain with Sender Policy Framework (SPF) DNS TXT records. 
```

Comment:
This prereq item has quite a bit of overlap with the next one - I'm not certain why we'd need both called out separately.